app/views/merchant/dashboard/index.html.erb

```diff
@@ -7,9 +7,13 @@
 <%= content_tag :p, "#{@merchant.city}, #{@merchant.state}" %>
 <%= content_tag :p, @merchant.zip %>
 
-<ul>
-  <% @merchant.item_orders.each do |item_order| %>
-  <li><%= content_tag :p, item_order.item.name %></li>
-  <%= button_to "Fulfill", "/merchant/", method: :patch, id: "item_order-#{item_order.id}", params: {item_order_id: item_order.id} %>
+<section id="pending-orders">
+  <%= content_tag :h3, "Pending Orders" %>
+
+  <% @merchant.distinct_orders.each do |order| %>
+    <%= link_to "Order Number #{order.id}", "/merchant/orders/#{order.id}" %>
+    <%= content_tag :p, "Created: #{order.created_at}" %>
+    <%= content_tag :p, "Total items in order: #{@merchant.quantity_per_merchant(order)}" %>
```

Comment:
It's amazing to me that the ONE line of code to get total items took so much time and effort 🧇